resources/maskinporten_clients.py

```diff
@@ -171,26 +175,36 @@ def create_client_key(
 
     if send_to_aws:
         try:
-            ssm_params = send_secrets(
-                secrets={
-                    "keystore": keystore,
-                    "key_id": key_id,
-                    "key_password": key_password,
-                },
-                maskinporten_client_id=client_id,
-                destination_aws_account_id=body.destination_aws_account,
-                destination_aws_region=body.destination_aws_region,
+            secrets_client = ForeignAccountSecretsClient(
+                body.destination_aws_account,
+                body.destination_aws_region,
+                client_id,
             )
-        except AssumeRoleAccessDeniedException as e:
+        except AssumeRoleAccessDeniedError as e:
             raise ErrorResponse(status.HTTP_422_UNPROCESSABLE_ENTITY, str(e))
 
```

Comment:
Skulle ikke HTTP error code være access denied? 403 Forbidden ?

Reply:
Tanken var at siden det ikke er vårt API som nekter tilgang, men en feil konfigurasjon i teamets egen AWS-konto, så syntes jeg det passet med 422s "unable to process the contained instructions". Men kanskje 403 er mer riktig? Har du noen tanker @petterhj?

Reply:
Hvis vi gir `403` indikerer vi vel at brukeren ikke har tilgang til å kalle ressursen, som vel ikke vil stemme helt nei? (I og med at det er vårt API som ikke har tilgang) (Om det er 422 som er det riktigste, er vel et annet spørsmål, men har ingen motforestillinger, egentlig, bare vi returnerer en forklarende nok feilmelding.

Reply:
Da tror jeg vi beholder den som det er. Feilmeldingen kommer fra AWS og er ganske detaljert i dette tilfellet (noe a la "User: xxx is not authorized to perform: sts:AssumeRole on resource: arn:aws:iam::xxx:role/dataplatform-maskinporten").